.github/workflows/push.yml

```diff
@@ -75,16 +73,18 @@ jobs:
         uses: actions/checkout@v3
 
       - name: 🚢 Deploy
-        uses: ./.github/actions/deploy
+        uses: agrc/cloud-run-docker-deploy-composite-action@v1
         with:
           web-api-key: ${{ secrets.WEB_API_KEY }}
```

Comment:
**quibble**: this seems extraneous

Reply:
That was left over from the local action. Thanks for catching that!